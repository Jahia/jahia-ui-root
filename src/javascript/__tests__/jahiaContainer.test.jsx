import React from 'react';
import renderer from 'react-test-renderer';
import JahiaContainer from '../Jahia';

describe('Jahia container', () => {

    test('full app', () => {
        const jahiaCtx = { locale: "en", uilang: "en", siteKey: "systemsite" };
        window.contextJsParameters = { user: {fullname: "mock_fullname", email: "mock_email"}, links: { whatsNew: "mock_url", documentation: "mock_url"}};
        const component = renderer.create(
            <JahiaContainer jahiaCtx={jahiaCtx}/>,
        );

        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
});
