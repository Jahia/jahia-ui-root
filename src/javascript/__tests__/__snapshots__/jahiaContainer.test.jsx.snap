// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jahia container full app 1`] = `""`;
